aten/src/ATen/native/quantized/cpu/fbgemm_utils.cpp

```diff
@@ -212,16 +212,20 @@ Tensor ConvertToChannelsLast3dTensor(const Tensor& src) {
 
 template <int kSpatialDim = 2>
 CAFFE2_API torch::jit::class_<ConvPackedParamsBase<kSpatialDim>> register_conv_params() {
+  // Note: SerializationType order should be fixed.
+  //       See onnx/unpack_quantized_weights.cpp
   using SerializationType = std::tuple<
-    at::Tensor,
-    c10::optional<at::Tensor>,
+    at::Tensor /*weight*/,
+    c10::optional<at::Tensor> /*bias*/,
     // these are meant to be torch::List<int64_t> but
     // it's not supported by onnx, so we'll use Tensor as
     // a workaround
-    torch::List<at::Tensor>,
-    torch::List<at::Tensor>,
-    torch::List<at::Tensor>,
-    at::Tensor>;
+    torch::List<at::Tensor> /*stride*/,
+    torch::List<at::Tensor> /*padding*/,
+    torch::List<at::Tensor> /*dilation*/,
+    at::Tensor /*groups*/,
+    at::Tensor /*transpose*/,
+    torch::List<at::Tensor> /*output_padding*/>;
```

Comment:
This breaks backward compatibility:

```
Jun 11 08:45:59 The PR is introducing backward incompatible changes to the operator library. Please contact PyTorch team to confirm whether this change is wanted or not. 
Jun 11 08:45:59 
Jun 11 08:45:59 Broken ops: [
Jun 11 08:45:59 	__getstate__(__torch__.torch.classes.quantized.Conv2dPackedParamsBase _0) -> ((Tensor, Tensor?, Tensor[], Tensor[], Tensor[], Tensor) _0)
Jun 11 08:45:59 	__setstate__(__torch__.torch.classes.quantized.Conv2dPackedParamsBase _0, (Tensor, Tensor?, Tensor[], Tensor[], Tensor[], Tensor) _1) -> (None _0)
Jun 11 08:45:59 	__getstate__(__torch__.torch.classes.quantized.Conv3dPackedParamsBase _0) -> ((Tensor, Tensor?, Tensor[], Tensor[], Tensor[], Tensor) _0)
Jun 11 08:45:59 	__setstate__(__torch__.torch.classes.quantized.Conv3dPackedParamsBase _0, (Tensor, Tensor?, Tensor[], Tensor[], Tensor[], Tensor) _1) -> (None _0)
Jun 11 08:45:59 ]
```

Unfortunately I don't think there's going to be an easy way around this. What you might have to do is re-bind ConvPackedParams as a new type. When you do that, I suggest making the `SerializationType` generic like in packed RNN cells: https://github.com/pytorch/pytorch/blob/eace0533985641d9c2f36e43e3de694aca886bd9/aten/src/ATen/native/RNN.cpp#L52

That way you can change the specific data that's encoded freely

Reply:
Any idea how to repro this locally? I am running `python test/backward_compatibility/check_backward_compatibility.py --new-schemas ~/Desktop/new_schemas.txt`, with `new_schemas.txt` created by the `dump_all_function_schemas.py`. But when I run it locally, it just says `Found backward compatible schemas for all existing schemas`

Reply:
Temporarily resolving it -- will be fixed when #40003 is updated and landed